### ISSUE TITLE
a question about UpsamplingBottleneck block 

### ISSUE THREAD

**Issue by cedricgsh** [davidtvs/PyTorch-ENet#50]
@davidtvs 
The official code does not have any activation functions in 1x1 expansion block in all Bottleneck blocks.
The paper said that they place Batch Normalization and PReLU **between all convolutions, not after all convolutions**
This is official code fragment in Bottleneck block.  I cannot find any activation functions after the region marked in red.
I hope you can check this problem.
![1613355171(1)](https://user-images.githubusercontent.com/33807572/107898697-5f900f00-6f77-11eb-82e3-0a10272e3b4d.png)

**Comment by davidtvs** [comment]
There's another activation on the return statement. See https://github.com/e-lab/ENet-training/blob/master/train/models/decoder.lua#L96

**Comment by cedricgsh** [comment]
Thanks for your answer. Maybe my expression is not very clear, so you are not making sense.
The activation on the return statement is marked in red. However, I mean, you add a needless PReLU after 1x1 expansion block, marked in blue.
![1613438077(1)](https://user-images.githubusercontent.com/33807572/108007905-05a44d80-703a-11eb-83dd-912b4c714982.png)

**Comment by davidtvs** [comment]
You are correct. I see it now. I'll make the change in the repository when I manage to retrain the corrected models.

Thanks for raising this to my attention!

**Comment by davidtvs** [comment]
Fixed now. Thanks!
